ftl/common/args.py

```diff
@@ -116,6 +116,13 @@ def base_parser():
         action='store',
         default=None,
         help='The port to expose for the dockerimage')
+    parser.add_argument(
+        '--log-path',
+        dest='log_path',
+        action='store',
+        default='/tmp',
```

Comment:
I think this actually might fail for a lot of systems because `/tmp` is usually restricted, can we default this to `/var/log` instead